os/include/tinyara/net/dns.h

```diff
@@ -59,7 +59,7 @@
  */
 
 /**
- * @file dns.h
+ * @file tinyara/net/dns.h
```

Comment:
why do we need tinyara folder?

Reply:
That is a Tinyara specific file, not a standard so that that location is tinyara/net. When user want to include that, he should include that path like below.
```
#include <tinyara/net/dns.h>
```